bootstrap/sql/migrations/native/1.3.1/postgres/schemaChanges.sql

```diff
@@ -0,0 +1,178 @@
+-- PostgreSQL DDL script for the Quartz database
+DROP TABLE IF EXISTS QRTZ_FIRED_TRIGGERS;
```

Comment:
do we need drop statements?, if we run by force this will unnecessarily drop those tables no?

Reply:
removed them , thanks :)